lib/ignored-paths.js

```diff
@@ -94,7 +94,11 @@ function relativize(globPattern, relativePathToOldBaseDir) {
     const globWithoutPrefix = globPattern.replace(/^!/, "");
 
     if (globWithoutPrefix.startsWith("/")) {
-        return `${prefix}/${relativePathToOldBaseDir}${globWithoutPrefix}`;
+        const relativePathPrefix = (typeof process !== "undefined" && process.platform === "win32")
+            ? relativePathToOldBaseDir.replace(/\\/g, "/")
+            : relativePathToOldBaseDir;
+
+        return `${prefix}/${relativePathPrefix}${globWithoutPrefix}`;
```

Comment:
I found `relativePathToOldBaseDir` can be `../../../foo/bar`, is this a problem?

Reply:
Hmm, it might be. I had assumed that one of `this.ignoreFileDir` and `this.options.cwd` would be the parent of the other. If `this.ignoreFileDir` and `this.options.cwd` are in different directories, maybe `this.getBaseDir()` should return a common ancestor.